### PR TITLE
Add a new docs file of M2 GPS device

### PR DESCRIPTION
**Pull request by lishuanghua**
Add a new docs file of M2 GPS device which supports static initialization.

### CODE REVIEW

File: modules/drivers/gnss/parser/newtonm2_parser.cc [hunk 1]
```diff
@@ -0,0 +1,918 @@
...
+bool NewtonM2Parser::HandleRawImu(const newtonm2::RawImu* imu) {
+  double gyro_scale = 0.0;
+  double accel_scale = 0.0;
+  float imu_measurement_span = 1.0 / 200.0;
+
+  if (newtonm2::is_zero(gyro_scale_)) {
+    newtonm2::ImuParameter param = newtonm2::GetImuParameter(imu_type_);
+
+    if (newtonm2::is_zero(param.sampling_rate_hz)) {
+      AERROR_EVERY(5) << "Unsupported IMU type ADUS16488.";
```
Comment (mickeyouyou) [inline modules/drivers/gnss/parser/newtonm2_parser.cc]: wrong error message "Unsupported IMU type ADUS16488"

File: docs/specs/Navigation/Newton-M2_Installation_Guide_cn.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+####修改定位模式
+在apollo/modules/localization/conf/localization_config.pb.txt文件中把配置localization_type: RTK改为localization_type: MSF。
+
+###$修改脚本
+在apollo/scripts/docker_adduser.sh文件中将#setup GPS device下面添加如下代码
+```
+if [ -e /dev/ttyACM0 ]; then
+  chmod a+rw /dev/ttyACM0
+fi
```
[the remark this answers is not in the document]
Comment (lishuanghua) [reply docs/specs/Navigation/Newton-M2_Installation_Guide_cn.md]: The code is already uploaded by the last pull request.

File: docs/specs/Navigation/Newton-M2_Installation_Guide_cn.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+####修改配置
+
+- 修改/apollo/modules/drivers/gnss/proto文件夹下面的配置文件config.proto，将NEWTONM2_TEXT = 30;NEWTONM2_BINARY = 31;添加进message Stream的后面;
+- 修改/apollo/modules/drivers/gnss/conf文件夹下面的配置文件gnss_conf.pb.txt，将gnss_conf_newton.pb.txt的内容全部拷贝覆盖gnss_conf.pb.txt的内容即可。
+
+####关闭雷达
+在apollo/modules/localization/conf/localization.conf文件中将：--enable_lidar_localization=true修改为：--enable_lidar_localization=false.
+
+####修改定位模式
+在apollo/modules/localization/conf/localization_config.pb.txt文件中把配置localization_type: RTK改为localization_type: MSF。
```
[the remark this answers is not in the document]
Comment (diegohu) [reply docs/specs/Navigation/Newton-M2_Installation_Guide_cn.md]: It should be highlighted since RTK mode is not supported in M2.

**Comment by mickeyouyou** [comment]
fix format  by `/home/travis/.cache/bazel/_bazel_travis/540135163923dd7d5820f3ee4b306b32/execroot/apollo/bazel-out/local-dbg/testlogs/modules/drivers/gnss/parser/newtonm2_parser_cpplint/test.log`

**Pull request closed without merging**